pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -311,3 +320,11 @@ func (s *Installer) CleanupLocalBoots(ctx context.Context, remove bool) error {
 	// finally, return an "already exists" error if boots exists and forceCleanup is not set
 	return errors.New("boots container already exists, delete the container manually or re-run the command with --force-cleanup")
 }
+
+func (s *Installer) url(originalURL string) string {
```

Comment:
Could we give this function a name that's a little more representative of what it does.